### ISSUE TITLE
Better default empty states with animation

### ISSUE THREAD

**Issue by shawnborton** [Expensify/App#12261]
Note: This issue is part of the greater **[Implement Our New Branding in NewDot](https://github.com/Expensify/Expensify/issues/239140)** project. 

**What is it:**
Bring life to our app by animating various empty states illustrations. We can split this project into two phases: first, let’s identify areas where we can use illustrations for empty states. Augenblick can help us create new illustrations. Then we can take those illustrations and bring them to life with animation. Again, Augenblick can help us by animating these illustrations. We can then use Lottie to implement the animations into the product.

![image](https://user-images.githubusercontent.com/2319350/198701714-872dda4b-7adb-4dc8-8cd5-39a4305b692f.png)

**Does it require a mini design doc:**
Yes

**Design finished:**
No

cc @GabiHExpensify

**Comment by shawnborton** [comment]
Adding a note that as part of [this conversation here](https://expensify.slack.com/archives/C01GTK53T8Q/p1667277018509469), we've decided to remove this one from the WAQ Project. That doesn't mean we shouldn't do this, but let's prioritize all other WAQ issues first before moving onto this one.

**Comment by roryabraham** [comment]
I personally haven't invested any time into this yet, but I'm definitely excited to work on it when the time is right. @shawnborton keep me posted on next steps and let me know how I can help!

We'll also want keep an eye on [this absolutely monstrous PR](https://github.com/lottie-react-native/lottie-react-native/pull/910) to add new architecture support in Lottie

**Comment by JmillsExpensify** [comment]
Ha @shawnborton what a coincidence. Check out the reviewer on that linked PR!

**Comment by shawnborton** [comment]
Hah, what are the odds! 

In other news, I'll post some ideas to the room today and we'll get kicked off with Augenblick next week.

**Comment by shawnborton** [comment]
Augenblick sent over some test animation Lottie files for us - they are mostly looking for freedback on if this format will work, and how the file size is. Thoughts @roryabraham ?

Files: [Expensify_LottieTests_112222.zip](https://github.com/Expensify/App/files/10071727/Expensify_LottieTests_112222.zip)

Given that the JSON file size is less than 100KB, I think this is going to work great. 

![runningTest](https://user-images.githubusercontent.com/2319350/203439790-f990a041-962b-4992-802c-c51b62b81b20.gif)

![ufoTest](https://user-images.githubusercontent.com/2319350/203439797-6ab32468-94b4-4f2c-beb9-72066b1186b8.gif)


![dollarTest](https://user-images.githubusercontent.com/2319350/203439775-8b527062-84f8-4609-8573-3db75a4153b3.gif)

**Comment by shawnborton** [comment]
I don't think we have an existing screen to use for this yet, we'd be making new layouts as part of this project.

**Comment by JmillsExpensify** [comment]
Shouldn't this issue be on hold for WAQ?

**Comment by shawnborton** [comment]
We discussed this previously and agreed that it makes sense to get started with Augenblick on this because we can do that in parallel to WAQ without needing Engineering resources. That being said, I can see an argument for Rory to hold off on doing any kind of engineering testing here given that he should be prioritizing WAQ. I still think it makes sense for us to be doing this work with Augenblick in the meantime though.

**Comment by roryabraham** [comment]
Haven't invested any time in this yet, but I'm excited to do so when the time comes!

**Comment by JmillsExpensify** [comment]
Ok cool, yeah totally agree we can push forward with Augenblick, I just don't think we should push forward with any engineering time until we're in a better spot with WAQ. I'm throwing a HOLD on this issue to make sure that's clear, though please feel free to use the issue for on-going Augenblick updates.

**Comment by roryabraham** [comment]
Still on HOLD (for my part, anyways...)

**Comment by shawnborton** [comment]
Not overdue. We have all of our animations and Lottie files completed, we just need to wait for WAQ to be finished and then we can hopefully start working through this one. 

Some treats for you:
![Expensify-Magic-Code-121222-kjs-1](https://user-images.githubusercontent.com/2319350/208447148-d7ad0412-d929-4167-9511-3d9d27133eed.gif)

![Expensify-Magic-Link-120922-kjs-1](https://user-images.githubusercontent.com/2319350/208447164-bcc4bf15-eb9b-4ca1-aa85-ca90cce140a1.gif)

![Expensify-Fast-Money-2-121322-kjs-1](https://user-images.githubusercontent.com/2319350/208447248-e34851da-28b3-4d9d-ac70-e77f3c2d4b49.gif)

**Comment by JmillsExpensify** [comment]
We're getting real close on the bugs. The bigger blockers for WAQ are going to be the headline initiatives, so once we get those out of the way we'll be back at it!

**Comment by shawnborton** [comment]
Not overdue, still holding.

**Comment by roryabraham** [comment]
Still on HOLD

**Comment by roryabraham** [comment]
Let's take this off hold!

**Comment by shawnborton** [comment]
Woo! I think a good place to start is first just getting Lottie to work in the app by adding a small animation to the About us page, basically replacing the "E" iconmark with the rotating coin animation we have:

![Expensify-Loading-120822-kjs-1](https://user-images.githubusercontent.com/2319350/211747919-c9c294f5-952b-4da1-ac86-392c165f88ca.gif)

**Comment by JmillsExpensify** [comment]
Nice one! Is Lottie compatible with Fabric yet?

**Comment by roryabraham** [comment]
> they are mostly looking for feedback on if this format will work, and how the file size is

Sorry I missed this question earlier. I'll see if I run into any issues and report back.

> Is Lottie compatible with Fabric yet?

Not fully.  https://github.com/lottie-react-native/lottie-react-native/pull/910 was merged but it looks like they're still working on Fabric support for iOS: https://github.com/lottie-react-native/lottie-react-native/pull/955

Regardless, since our app isn't on Fabric yet this isn't really a blocker for the moment.

**Comment by roryabraham** [comment]
iOS POC. Performance on the simulator is okay but noticing a bit of choppiness ... curious to see if it runs any smoother on a physical device.

https://user-images.githubusercontent.com/47436092/212443993-e7837759-84d5-46af-a3bc-ecdb2287afa8.mov

**Comment by shawnborton** [comment]
That does look a bit choppy. Also, how are you determining the correct size of the animation? I would think it would be bigger (bigger than the current 52px icon that is there)

**Comment by roryabraham** [comment]
> Also, how are you determining the correct size of the animation?

I just passed 80 for the width and height, which [matches what we have today](https://github.com/Expensify/App/blob/2189f4ecc699e341cda4ce75a0e7d6fdb1c45e69/src/pages/settings/AboutPage/AboutPage.js#L81)

**Comment by roryabraham** [comment]
Posted a proposal related to this issue in WN: https://expensify.slack.com/archives/CC7NECV4L/p1674010182488429

**Comment by tomekzaw** [comment]
Let me take a look at this.

**Comment by roryabraham** [comment]
While @tomekzaw's investigating ways to improve lottie performance, I'm going to continue getting a cross-platform implementation set up in our app

**Comment by JmillsExpensify** [comment]
I was catching up earlier, I think think we discovered that with minor tweaks our performance issues will likely be solved. Is that right?

**Comment by cristipaval** [comment]
Should we also include Preferences page in this issue?
![image](https://user-images.githubusercontent.com/18078393/215056057-f373ae4d-f2b9-4353-a6d7-aec966538348.png)

**Comment by shawnborton** [comment]
Yup, I agree with that. 

I think to start, we should focus on all of the pages in Settings. After that, we can take a look at the sign up flow, especially once passwordless is finished. 

So that would be this:
![image](https://user-images.githubusercontent.com/2319350/215092169-cfcb118e-27b6-4508-a22b-bbdcd4775715.png)


And I think we can get some new animations to use for Workspaces and Preferences.

**Comment by JmillsExpensify** [comment]
> I think to start, we should focus on all of the pages in Settings. After that, we can take a look at the sign up flow, especially once passwordless is finished.

Love this idea.

**Comment by shawnborton** [comment]
Augenblick will get started on the additional animations on 2/20. @roryabraham - how are things going on the Lottie front?

**Comment by roryabraham** [comment]
@shawnborton just finished getting Lottie up-and-running on all platforms. It ended up being quite simple, which is great. I did open up [one upstream PR](https://github.com/react-native-web-community/react-native-web-lottie/pull/78), but it's non-critical.

**Comment by roryabraham** [comment]
Okay, I tried to switch over from `.json` files to `.lottie` files but it's not as simple as I thought because the libraries I was using don't yet support them (bummer because they're only about 10% the size of the `.json` equivalent).

Anyways, I'm going to put up a PR to swap out our existing `.gifs` using the `.json` files and then create a follow-up issue to use `.lottie` files once they're supported. I started looking into what it would take to bring support into those libraries and it seems pretty doable.

**Comment by shawnborton** [comment]
Amazing! Please tag me when you fire up the PR, can't wait to get these animations in there!

**Comment by roryabraham** [comment]
- Initial PR is up: https://github.com/Expensify/App/pull/14393
- Follow up issue for `.lottie` files: https://github.com/Expensify/App/issues/15053

**Comment by shawnborton** [comment]
Now that we have Lottie merged, I think you could get a head start on creating the colored top areas of the various settings pages:
![image](https://user-images.githubusercontent.com/2319350/221002102-f7969a50-921c-48b6-83ae-deea37ae7298.png)


I think the About page is a good place to start, and you can use the coin lottie animation [here](https://www.dropbox.com/s/ik8j2zqbiimj762/Expensify-Loading-120822-kjs-1.json?dl=0).

**Comment by roryabraham** [comment]
Got a draft PR up: https://github.com/Expensify/App/pull/15462

The biggest challenge so far is syncing the status bar color with the current navigation page. I do think it could be simplified if we HOLD on the react-navigation revamp, but that might not be necessary.

**Comment by MelvinBot** [comment]
Looks like something related to `react-navigation` may have been mentioned in this issue discussion.

As a reminder, please make sure that all proposals are not workarounds and that any and all attempt to fix the issue holistically have been made before proceeding with a solution. Proposals to change our `DeprecatedCustomActions.js` files should not be accepted.

Feel free to drop a note in [#expensify-open-source](https://expensify.slack.com/archives/C01GTK53T8Q) with any questions.

**Comment by roryabraham** [comment]
Seems related to https://github.com/Expensify/App/issues/13001

**Comment by roryabraham** [comment]
Hey @shawnborton, the remaining technical portion of this issue (i.e: implementing the two-tone layouts shown in [this comment](https://github.com/Expensify/App/issues/12261#issuecomment-1442266992) is blocked on https://github.com/Expensify/App/issues/13001.

Currently, we have a hard-coded dark green status bar on Android, but we want the bright colors from those screenshots to extend to the top of the screen. I spent almost two whole days trying to get that working on Android last week, but it turned out to be challenging to do without breaking the keyboard-avoiding layouts we have in place. I've requested help from Margelo to move things along, but for now [this PR](https://github.com/Expensify/App/pull/15462) will have to go on HOLD

**Comment by shawnborton** [comment]
Sounds good, keep me posted on how that goes!

**Comment by shawnborton** [comment]
Looks like we're making good progress on the status bar issue!

**Comment by roryabraham** [comment]
Yep, @shawnborton just left a review on the status bar PR. Seems close now, so I'm going to pick this back up and work on getting it completed on iOS.

**Comment by roryabraham** [comment]
Picking this back up this week

**Comment by shawnborton** [comment]
@roryabraham what is the status here?

**Comment by roryabraham** [comment]
Sorry for the delayed reply @shawnborton, removed the reviewing label here. Unfortunately the status bar fix was reverted, so this is technically back on HOLD for https://github.com/Expensify/App/pull/15778, which is why it has been off my radar.

**Comment by shawnborton** [comment]
Is there a reason why we can't make forward progress on this while we wait for the status bar to be fixed? Or what does it look like if we implement this but the status bar still remains dark at the very top? And that's only for Android, right?

I know @grgia was interested in this one too, so if you are finding that you don't have the capacity to work on this, perhaps we can hand it off?

**Comment by roryabraham** [comment]
> Is there a reason why we can't make forward progress on this while we wait for the status bar to be fixed?

We can get it polished up on iOS, for sure.

> Or what does it look like if we implement this but the status bar still remains dark at the very top?

You'll see a bright color such as orange in the header, then a thin strip of dark green in the status bar.
 
> that's only for Android, right?

Correct

**Comment by shawnborton** [comment]
Okay, I could live with that personally - or in the very least, I'd love to see a quick screenshot of it but I think it's going to look just fine.

**Comment by roryabraham** [comment]
In the Figma board, I see a few different header sizes:

|Small|Medium|Large|
|---|---|---|
|<img width="300" alt="image" src="https://user-images.githubusercontent.com/47436092/234024128-3398d274-81ed-4dee-b760-1743cd5ad339.png">|<img width="300" alt="image" src="https://user-images.githubusercontent.com/47436092/234023965-db035a54-5921-4dce-8cf3-93be21ff7231.png">|<img width="300" alt="image" src="https://user-images.githubusercontent.com/47436092/234023660-535dcfdb-f0d4-4666-a676-778d67b7ca5b.png">|

Is that intentional based on the screen, or do we want to standardize this a bit?

It looks like the small layout takes up 30% of the screen height, the medium takes up about 36% of the screen height, and the large takes up about 43% of the screen height.

**Comment by shawnborton** [comment]
Not sure where you are getting those, but I was previously working off of some designs that all used the same height:
<img width="1195" alt="image" src="https://user-images.githubusercontent.com/2319350/234026188-c49f0d7a-5482-413e-9ce0-a9683cdfe236.png">


I'll follow up here shortly with more specifics though on colors, animations, etc. 

I am thinking we also might want to punt the Payments page for now since we have a lot of changes coming there as part of the (former) N7/8 projects.

**Comment by JmillsExpensify** [comment]
Good call on the payments front. I'm not sure how the header would work with the Wallet balance, since we said the wallet balance would go at the top.

**Comment by roryabraham** [comment]
One thing I've run into is that some of the Lottie animation files I have seem to have some built-in padding and background color. We should try to ensure that there's no extra padding "baked in" to the lottie animation.

**Comment by roryabraham** [comment]
One example is `Dollar.json` in the [draft PR](https://github.com/Expensify/App/pull/15462)

**Comment by shawnborton** [comment]
Looks like there are no screenshots in your draft PR so I'm not sure exactly what you are referring to. Can you share a quick screenshot here when you get a moment? Thanks!

**Comment by roryabraham** [comment]
Sure, in this screenshot, I have put a red border on a view surrounding the lottie animation. This view has no other styles except:

```js
{
    justifyContent: 'center',
    alignItems: 'center',
}
```

<img width="349" alt="image" src="https://user-images.githubusercontent.com/47436092/234088042-22d6ffe7-c732-48f8-89e0-e3eca5fe4c8b.png">

The view is taking up the full width which I think is expect in the flex layout I have set up, but you can see there's additional vertical padding above and below the animation that shouldn't be there. It seems like this padding is coming from the Lottie file itself.

**Comment by shawnborton** [comment]
Interesting, I don't think we're planning to use that animation? I am not even sure where it came from. 

That being said, I think the best thing to do might be to ask Augenblick to setup all animations in the same frame size, say 375x220. Then we can just make sure we scale that proportionally, so if we have a wider phone then it would go up to 390x229 for instance.

Does that sound like a good plan? This way everything is nice and consistent from an implementation standpoint.

**Comment by shawnborton** [comment]
@roryabraham can you try these? 

[ExpensifyLottieFiles_042532.zip](https://github.com/Expensify/App/files/11327191/ExpensifyLottieFiles_042532.zip)

The idea is that they will take up the whole width of that area under the page header. For wider phones, we should just make the animation grow proportionally, so we essentially just want to give it a width of 100% and the height should auto adjust. Let me know if that makes sense!

**Comment by roryabraham** [comment]
Thanks @shawnborton!

**Comment by roryabraham** [comment]
I'm going to start implementing what I see in [these screenshots](https://github.com/Expensify/App/issues/12261#issuecomment-1520268439), but I'll keep an eye out here if you want to provide a complete list of screen/color/animation combinations.

**Comment by shawnborton** [comment]
Sure, so maybe we start with these 4? I made some color changes as a heads up:
<img width="734" alt="image" src="https://user-images.githubusercontent.com/2319350/234626031-de57f18d-8e2f-4455-85da-1b0486434699.png">


So here are the background values:
- Workspaces: `pink800` = `#49225B`
- Preferences: `blue500` = `#0676DE`
- Security: `ice500` = `#4ED7DE`
- About: `yellow600` = `#D18000`

**Comment by shawnborton** [comment]
Actually maybe we can do the payments empty state? @kevinksullivan perhaps you can confirm, but I think we can still have an empty state up until any of the following happen:
- A payment method gets added, which spawns the top wallet section with a balance as well as a bank accounts section
- A card gets assigned, which spawns the assigned cards section

![image](https://user-images.githubusercontent.com/2319350/234630026-2327ef59-0d38-405a-b494-29f601b42348.png)

**Comment by roryabraham** [comment]
@shawnborton should we also do the initial settings page? Since the header content in that one is a bit different from the other pages (with an avatar image and some other stuff), it would be good to make sure that the new layout works in that case too.

**Comment by shawnborton** [comment]
By initial settings page, do you mean this where we have the slightly different color header? 

![image](https://user-images.githubusercontent.com/2319350/234654223-ace5cfa5-bd8a-4e33-a6ff-e562d6a19cc3.png)

I would be down for that. We should try to make that a reusable component because we'll also want to do that for things like viewing a profile, viewing room details, etc. 


Here is the missing lottie animation: [WorkspaceLottie_042623.zip](https://github.com/Expensify/App/files/11335679/WorkspaceLottie_042623.zip)

**Comment by roryabraham** [comment]
Yeah, that's the one @shawnborton. I noticed that the background color appears to be `#11261f`, which isn't in `colors.js` right now. What should I call that? `green900`?

**Comment by shawnborton** [comment]
That should be in there as the same color we use for the LHN and preview cards. The value should be `#07271F`

Technically it is our `green800` value too.

**Comment by shawnborton** [comment]
Actually let's add it in as `green800` and it should be `#002E22;`

**Comment by shawnborton** [comment]
Actually sorry I take that back. Let's just try to use the same color we use for the LHN/preview cards/etc - this way it translates really well to changing for light mode.

**Comment by roryabraham** [comment]
Sounds good. Any concerns about the contrast here?

<img width="409" alt="image" src="https://user-images.githubusercontent.com/47436092/234669589-98aa3f06-270b-4e1a-822d-6f05e4c4fdde.png">

**Comment by roryabraham** [comment]
Here's how that looks:

<img width="327" alt="image" src="https://user-images.githubusercontent.com/47436092/234670779-156b7c01-9475-459c-876c-50913c736926.png">

**Comment by shawnborton** [comment]
Thoughts? If that's not enough contrast, we can try `ice600` which is `#4BA6A6`

**Comment by roryabraham** [comment]
Here's how that looks:

<img width="381" alt="image" src="https://user-images.githubusercontent.com/47436092/234671929-bae912cb-fbf2-4292-8252-f0e376f69e05.png">

I think it looks a bit more dull than `ice500`, not as much of a fan. Maybe we could just change the text color to something darker, like `ice800`, which look like this:

<img width="392" alt="image" src="https://user-images.githubusercontent.com/47436092/234673141-26ce7582-a978-4779-bafd-603b91777084.png">

One downside there is that the status bar icons are still light, which we could fix on a screen-by-screen basis but might be a bit of a pain.

What do you think is best? We could also just continue with white on `ice500` 🤷🏼

**Comment by shawnborton** [comment]
I'm down to use white on ice500 and go from there.

Also, looks like I might need to ask Augenblick to add some negative space below the animation. I'll wait until you have them all ready to preview and then I can go back to them with feedback.

**Comment by JmillsExpensify** [comment]
Interesting! I like Shawn's idea to just show a nice full-screen empty state for Payments if no bank account or Expensify Card exists. While we're at it, we should do what we've been talking about for a long time and change the name of this row back to `Wallet`.

**Comment by kevinksullivan** [comment]
Just catching up to answer @shawnborton here

>A payment method gets added, which spawns the top wallet section with a balance as well as a bank accounts section
Correct

>A card gets assigned, which spawns the assigned cards section

This would also spawn the bank accounts section.

**Comment by kevinksullivan** [comment]
>While we're at it, we should do what we've been talking about for a long time and change the name of this row back to Wallet.

@JmillsExpensify this was planned in the [N7 card doc](https://docs.google.com/document/d/1iELOXPOAcnuIPZJ3XbO1S8_3qHaM31Ptf8dR4LRZMyE/edit#heading=h.ns12dr1f35rx), but I agree we could pull it out since that scope is going to change again. Want me to just create a separate issue for it?

**Comment by roryabraham** [comment]
The Wallet page is a bit more involved because of how the PaymentsPages is currently implemented, so doing _just_ the empty state would be more work than just reimplementing the PaymentsPage as the WalletPage. Not a huge lift, I just need to see the rest of that mockup. Not opposed to tackling it in a follow-up issue + PR though.

**Comment by shawnborton** [comment]
Cool, that works for me if you want to do it as a follow up.

**Comment by JmillsExpensify** [comment]
Same, that works.

**Comment by roryabraham** [comment]
The PR is finally ready for review

**Comment by roryabraham** [comment]
PR updated

**Comment by roryabraham** [comment]
Ready to take this over the finish line. Latest discussion in slack [here](https://expensify.slack.com/archives/C03KN50J0PM/p1685986293650929?thread_ts=1683853093.009779&cid=C03KN50J0PM)

**Comment by roryabraham** [comment]
The react-navigation reboot kind of blew up my PR, so I'm going to take a more incremental approach and focus on getting smaller chunks actually merged so that each PR can move faster.

**Comment by roryabraham** [comment]
Got another partial PR up and ready for review: https://github.com/Expensify/App/pull/20663

**Comment by shawnborton** [comment]
Amazing! How is progress going on the top animated portion?

**Comment by roryabraham** [comment]
Hopefully will progress faster next week – I got caught up with deploy fires in the second half of this week. Got [this PR](https://github.com/Expensify/App/pull/20663) fixed up and ready for another round of reviews.

**Comment by roryabraham** [comment]
Sorry for the OOO-related delay @shawnborton. This is back at the top of my priority list on Monday

**Comment by roryabraham** [comment]
Preferences page was merged today, so next up I think I can to work on a few PRs in parallel for other pages.

I think I know what's needed for `workspaces` and `security` (because I already have mockups), so I'll start with those two.

**Comment by shawnborton** [comment]
Amazing, let's keep up the momentum! 

One thing I noticed is that the stroke weight feels a bit thin with the preferences page. Is there a way to ensure the stroke stays at exactly 1px thick, regardless of screen size?

**Comment by roryabraham** [comment]
@shawnborton sorry, can you help clarify the stroke weight you're referring to? Are you talking about stroke weight in the lottie animation itself?

**Comment by shawnborton** [comment]
Yup exactly, all of the strokes found in the animated portion. Those ideally should be at 1px.

**Comment by roryabraham** [comment]
I can check if it's something I broke switching the under-the-hood renderer from svg to canvas, but other than that the stroke is not something I'm controlling at the JS level. It may be intrinsic to the animation file itself. I'll look into it

**Comment by roryabraham** [comment]
Hey @shawnborton, I tried reverting the SVG => canvas change to see if it had any effect on the stroke, but it seems not. I've also tried opening the PreferencesDJ.json file to search for a stroke field but it's pretty compressed / incomprehensible. I think we need to go back to Augenblick and ask for them to hardcode or increase the stroke weight in the animation file.

**Comment by roryabraham** [comment]
There may be more info [here](https://github.com/airbnb/lottie-web/issues/1294). Specifically [this comment](https://github.com/airbnb/lottie-web/issues/1294#issuecomment-596039943)? Though I'm not sure this is the same problem because we aren't really resizing the animation much

**Comment by roryabraham** [comment]
WorkspacesListPage PR should hopefully be merged tomorrow. Removing the reviewing label so this will go red in my K2 without updates.

Also, I'll be meeting with a lottie-react-native maintainer on Wednesday to discuss a few updates.

**Comment by JmillsExpensify** [comment]
These all look amazing! Keep it coming.

**Comment by roryabraham** [comment]
Ok, so I think next up I'll do `InitialSettingsPage` and `AboutPage` since they are a bit different in that there's some additional content in the header, not just an image.

Once we're through that, I'll open this up to contributors for a number of smaller ($250) jobs to crank out the rest of the pages in separate PRs using the layouts / patterns I've built so far. Sound good?

Other TODOs for me:

- Will continue cracking away at performance (dotLottie support, created temporary improvement [here](https://github.com/Expensify/App/pull/23776))
- Want to make sure I loop back to standardize how we're animating the StatusBar. Two different solutions were merged at almost the same time so we should button that up.

**Comment by shawnborton** [comment]
Works for me. Let's keep the momentum up and knock these all out!

**Comment by roryabraham** [comment]
DotLottie file is available in lottie-react-native as of 6.1.0: https://github.com/lottie-react-native/lottie-react-native/releases !

**Comment by roryabraham** [comment]
Picking this back up, going to work on switching to `.lottie`

**Comment by shawnborton** [comment]
Not overdue. @roryabraham I think you said you were turning your attention back here soon, what are the next steps?

**Comment by roryabraham** [comment]
Creating some external issues to build out these animations:

- https://github.com/Expensify/App/issues/26775

**Comment by roryabraham** [comment]
Created a separate issue to track enabling lottie file format: https://github.com/Expensify/App/issues/26857

**Comment by roryabraham** [comment]
We're working in parallel on the security and about pages, as well as DotLottie file formats. @shawnborton

Also created https://github.com/Expensify/App/issues/27211

**Comment by roryabraham** [comment]
Created https://github.com/Expensify/App/issues/27407 – the final page for this issue

**Comment by roryabraham** [comment]
Just fixed the animation for the About page

**Comment by roryabraham** [comment]
So the last piece here is the about page, and for some reason the animation just doesn't work normally on iOS Safari. We can hack a fix by using a canvas renderer instead of svg, but that messes up the heights. We should be able to hack a fix for the height, but I wonder if we might just ask Augenblick if they can see if there's something wrong with the animation in AfterEffects

**Comment by roryabraham** [comment]
Maybe we hack a fix then create a follow-up

**Comment by roryabraham** [comment]
Because the last holdout here is the About page, and the About page is being handled in https://github.com/Expensify/App/issues/26779, I think we can close this out! We're also implementing more improvements to our usage of lottie, but they're also being handled separately

**Comment by shawnborton** [comment]
Amazing!!
